src/pages/stop-details/stop-details.html

```diff
@@ -7,29 +7,39 @@
 
 <ion-header>
   <ion-navbar>
-    <ion-title align="center">{{this.stopName}}</ion-title>
-    <ion-buttons end>
-    <button ion-button clear icon-only class="googleMapPin" (click)="MapbuttonClick()">
-          <ion-icon ios="ios-pin" md="md-pin"></ion-icon>
-    </button>
-      <button ion-button clear icon-only class="favoriteButton" (click)="addFav(this.stopName)">
-                  <ion-icon ios="ios-star-outline" md="md-star-outline" *ngIf="!fav"></ion-icon>
-                  <ion-icon ios="ios-star" md="md-star" *ngIf="fav"></ion-icon>
-      </button>
-    </ion-buttons>
+    <ion-title align="center">Stop Details</ion-title>
   </ion-navbar>
 </ion-header>
 
+
 <ion-content>
-  <div class="time-display" align="center">
-    <h1 *ngIf="nextBuses.length > 0">{{this.nextBuses[0].showString}}</h1>
-    <h2 *ngIf="nextBuses.length > 0">{{this.nextBuses[0].line}}</h2>
-    <h1 *ngIf="nextBuses.length === 0">No upcoming buses</h1>
-    <h2 *ngIf="nextBuses.length === 0">Come back later</h2>
-  </div>
+  <ion-navbar class='tempTitle'>
+    <ion-title class='tempTitle' align="center">{{this.stopName}}</ion-title>
```

Comment:
we should give this field a separate css class from the navbar. then we should make the stop name here bigger than navbar font

Reply:
Also longer stop names get cut off from the screen. eg: pearson/miew van de rohe